console/src/components/Services/Events/Common/Components/InvocationLogsTable.tsx

```diff
@@ -206,6 +231,17 @@ const InvocationLogsTable: React.FC<Props> = props => {
     },
   });
 
+  const getNumOfPages = (
+    currentPageSize: number,
+    currentCount: number | undefined
+  ) => {
+    if (currentCount) {
+      return Math.ceil(currentCount / currentPageSize);
+    }
+
+    return currentPageSize;
```

Comment:
Don't understand. Why you return `currentPageSize` in a function that's supposed to return a number of pages?